### ISSUE TITLE
New version?

### ISSUE THREAD

**Issue by kloczek** [HewlettPackard/netperf#75]
https://github.com/HewlettPackard/netperf/compare/netperf-2.7.0...master shows +40 commits since laste release which was in 2015.
IMO it would be good to flush all those changesa and release neww version.

Do you have any plans to do that soon? 🤔

**Comment by vincele** [comment]
Yeah, especially since there are some compilation breakage that has been fixed.

**Comment by t-lo** [comment]
+1 for a new release, what can I do to help?
